tike/tomo.py

```diff
@@ -182,77 +142,23 @@ def reconstruct(
     """
     Lr = tomopy.recon(tomo=line_integrals.real,
                       theta=theta,
-                      algorithm=algorithm,
                       init_recon=obj.real,
-                      num_iter=niter, **kwargs,
+                      **kwargs,
```

Comment:
```init_recon``` is also an algorithm parameter. I'd also call this simply ```init```. Please don't ask why we have init_recon in TomoPy. :-) 

Also we can use ```algorithm=algorithm``` as an argument, but pass algorithm input parameters in a dictionary. Because, for simplicity the user can be able to call an algorithm without creating a dictionary.

Reply:
https://github.com/tomopy/tomopy/blob/68875a475744828b13e2aecafee82d5f2a247a7f/tomopy/recon/algorithm.py#L74-L76

`init_recon` is the name of the variable in TomoPy. I can't change it to `init` because it is part of the TomoPy interface.

You don't need a dictionary. Any keyword arguments you give to `tike.tomo.recon` are passed to `tomo.recon`.